coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
```

Comment:
- [ ] Nice work done on your project; kindly check I get this error while setting up your project, you should fix it in your gem file.

![error](https://user-images.githubusercontent.com/48371156/200091645-0f6ec61b-60c1-43f9-b031-b33627d49216.png)